### PR TITLE
Add/Modify new keywords for Ver.4.1 to 4.5

### LINKED ISSUE #187: Add new keywords for Ver.4.4 to 4.5.
## Enhancement
I want you to add new keywords for Ver.4.1 to 4.5.

So, basically, I referred to here and picked up the missing basic keywords

https://genshin-impact.fandom.com/wiki

We do not claim copyright in the following data: Feel free to use it.

### character
```json5
{
  "en": "Caterpillar",
  "ja": "キャタピラー",
  "zhCN": "卡特皮拉",
  "tags": [
    "fontaine",
    "character-sub"
  ]
},
{
  "en": "Lanoire",
  "ja": "ラノール",
  "zhCN": "莉诺尔",
  "tags": [
    "fontaine",
    "character-sub"
  ]
},
{
  "en": "Narzissenkreuz",
  "ja": "ナルツィッセンクロイツ",
  "zhCN": "纳齐森科鲁兹",
  "tags": [
    "fontaine",
    "character-sub"
  ]
},
{
  "en": "Callirhoe",
  "ja": "カリロエー",
  "zhCN": "卡莉露",
  "tags": [
    "mondstadt",
    "fontaine",
    "character-sub"
  ],
  "notes": "Ver4.1のイベント「流れゆく水に詩を紡いで」の純水精霊"
},
{
  "en": "Xianyun",
  "ja": "閑雲",
  "zhCN": "闲云",
  "pronunciationJa": "かんうん",
  "tags": [
    "liyue",
    "character-main"
  ],
  "notes": "留雲借風真君/三眼五顯仙人"
},
{
  "en": "Fujin",
  "ja": "浮錦",
  "zhCN": "浮锦",
  "pronunciationJa": "ふきん",
  "tags": [
    "liyue",
    "character-sub"
  ],
  "notes": "名もなき魔神眷属の仙人"
},
{
  "en": "Lingyuan",
  "ja": "霊淵",
  "zhCN": "灵渊",
  "pronunciationJa": "れいえん",
  "tags": [
    "liyue",
    "character-sub"
  ],
  "notes": "名もなき魔神眷属の仙人"
},
{
  "en": "Herblord",
  "ja": "薬君",
  "zhCN": "药君",
  "pronunciationJa": "やくくん",
  "tags": [
    "liyue",
    "character-sub"
  ],
  "notes": "名もなき魔神眷属の仙人"
},
{
  "en": "Uncle Luo",
  "ja": "羅おじ",
  "zhCN": "罗叔",
  "tags": [
    "liyue",
    "character-sub"
  ],
  "notes": "読みはおそらく「あみおじ」(要確認)"
},
{
  "en": "Little Mao",
  "ja": "マオくん",
  "zhCN": "小茂",
  "pronunciationJa": "まおくん",
  "tags": [
    "liyue",
    "character-sub"
  ]
},
{
  "en": "Shuyu",
  "ja": "漱玉",
  "zhCN": "漱玉",
  "pronunciationJa": "そうぎょく",
  "tags": [
    "liyue",
    "character-sub"
  ]
},
{
  "en": "Yuandai",
  "ja": "遠黛",
  "zhCN": "远黛",
  "pronunciationJa": "えんたい",
  "tags": [
    "liyue",
    "character-sub"
  ]
},
{
  "en": "Gaming",
  "ja": "嘉明",
  "zhCN": "嘉明",
  "pronunciationJa": "がみん",
  "tags": [
    "liyue",
    "character-main"
  ]
},
{
  "en": "Man Chai",
  "ja": "ウェンツァイ",
  "zhCN": "文仔",
  "pronunciationJa": "うぇんつぁい",
  "tags": [
    "liyue",
    "character-sub"
  ]
},
{
  "en": "Houzhang",
  "ja": "候章",
  "zhCN": "侯章",
  "pronunciationJa": "こうしょう",
  "tags": [
    "liyue",
    "character-sub"
  ],
  "notes": "削月築陽真君/三眼五顯仙人"
},
{
  "en": "Jiehu",
  "ja": "接笏",
  "zhCN": "接笏",
  "pronunciationJa": "せっしゃく",
  "tags": [
    "liyue",
    "character-sub"
  ],
  "notes": "理水畳山真君/三眼五顯仙人"
},
{
  "en": "Yip Tak",
  "ja": "葉徳",
  "zhCN": "叶德",
  "pronunciationJa": "ようとく",
  "tags": [
    "liyue",
    "character-sub"
  ]
},
{
  "en": "Chiori",
  "ja": "千織",
  "zhCN": "千织",
  "pronunciationJa": "ちおり",
  "tags": [
    "fontaine",
    "inazuma",
    "character-main"
  ]
}
```
### living-being
```json5
  {
    "en": "Redbill Pelican",
    "ja": "赤喙ペリカン",
    "zhCN": "朱喙鹈鹕",
    "pronunciationJa": "あかくちばしぺりかん",
    "tags": [
      "liyue",
      "living-being"
    ]
  },
  {
    "en": "Fluff-Fleece Goat",
    "ja": "フワフワヤギ",
    "zhCN": "柔柔羊",
    "pronunciationJa": "ふわふわやぎ",
    "tags": [
      "liyue",
      "living-being"
    ]
  },
  {
    "en": "Jadestone Turtle",
    "ja": "玉璜ガメ",
    "zhCN": "玉璜龟",
    "pronunciationJa": "ぎょくごうがめ",
    "tags": [
      "liyue",
      "living-being"
    ]
  },
  {
    "en": "Venerable Jadestone Turtle",
    "ja": "玉璜古ガメ",
    "zhCN": "玉璜古龟",
    "pronunciationJa": "ぎょくごうこがめ",
    "tags": [
      "liyue",
      "living-being"
    ],
    "note": "遺瓏埠北部の湾内に生息する巨大亀"
  },
  {
    "en": "Jadestone Turtle",
    "ja": "秋羽鴨",
    "zhCN": "秋绒鸭",
    "pronunciationJa": "しゅううかも",
    "tags": [
      "liyue",
      "living-being"
    ]
  },
  {
    "en": "Jade Heartfeather Bass",
    "ja": "ヒスイのハートフェザースズキ",
    "zhCN": "玉玉心羽鲈",
    "tags": [
      "liyue",
      "living-being",
      "item"
    ]
  },
  {
    "en": "Malachitin Lumibug",
    "ja": "玉光虫",
    "zhCN": "珉光虫",
    "pronunciationJa": "ぎょくこうちゅう",
    "tags": [
      "liyue",
      "living-being"
    ]
  }
```
### enemies
```json5
{
  "en": "Xuanwen Beast",
  "ja": "玄文獣",
  "zhCN": "玄文兽",
  "pronunciationJa": "げんもんじゅう",
  "tags": [
    "liyue",
    "living-being",
    "enemy"
  ]
},
{
  "en": "Solitary Suanni",
  "ja": "山隠れの猊獣",
  "zhCN": "隐山猊兽",
  "pronunciationJa": "やまがくれのげいじゅう",
  "tags": [
    "liyue",
    "enemy-boss"
  ]
}
```
## weapon
```json5
{
  "en": "Ultimate Overlord's Mega Magic Sword",
  "ja": "スーパーアルティメット覇王魔剣",
  "zhCN": "究极霸王超级魔剑",
  "tags": [
    "weapon",
    "claymore"
  ],
  "note": "Ver4.3のイベント「薔薇と銃士」報酬"
  "id": "ultimate-overlords-mega-magic-sword"
},
{
  "en": "Verdict",
  "ja": "裁断",
  "zhCN": "裁断",
  "tags": [
    "weapon",
    "claymore"
  ]
},
{
  "en": "Crane's Echoing Call",
  "ja": "鶴鳴の余韻",
  "zhCN": "鹤鸣余音",
  "tags": [
    "weapon",
    "catalyst"
  ]
},
{
  "en": "Uraku Misugiri",
  "ja": "有楽御簾切",
  "zhCN": "有乐御簾切",
  "tags": [
    "weapon",
    "sword"
  ]
},
{
  "en": "Dialogues of the Desert Sages",
  "ja": "砂中の賢者達の問答",
  "zhCN": "沙中伟贤的对答",
  "tags": [
    "weapon",
    "polearm"
  ],
  "note": "Ver4.5のイベント「巧みなる錬金経営」報酬"
}
```
### Item
```json5
  {
    "en": "Cloudseam Scale",
    "ja": "凝雲の鱗甲",
    "zhCN": "凝云鳞甲",
    "pronunciationJa": "ぎょううんのりんこう",
    "tags": [
      "drop-boss",
      "liyue"
    ],
    "notes": "山隠れの猊獣のドロップアイテム"

  },
  {
    "en": "Clearwater Jade",
    "ja": "清水玉",
    "zhCN": "清水玉",
    "pronunciationJa": "せいすいぎょく",
    "tags": [
      "drop",
      "liyue"
    ]
  },
  {
    "en": "Spirit Carp",
    "ja": "蒼霊の鯉",
    "zhCN": "苍灵鲤",
    "pronunciationJa": "そうれいのこい",
    "tags": [
      "item",
      "liyue"
    ]
  }

```
### Location
```json5
  {
    "en": "Yilong Wharf",
    "ja": "遺瓏埠",
    "zhCN": "遗珑埠",
    "pronunciationJa": "いろうふ",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Mt. Mingyuan",
    "ja": "暝垣山",
    "zhCN": "暝垣山",
    "pronunciationJa": "めいえんさん",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Qiaoying Village",
    "ja": "翹英荘",
    "zhCN": "翘英庄",
    "pronunciationJa": "ぎょうえいそう",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Mt. Lingmeng",
    "ja": "霊濛山",
    "zhCN": "灵濛山",
    "pronunciationJa": "れいもうさん",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Jademouth",
    "ja": "宝玦の口",
    "zhCN": "宝玦口",
    "pronunciationJa": "ほうけつのくち",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Chenlong Cleft",
    "ja": "沈瓏淵",
    "zhCN": "沉珑渊",
    "pronunciationJa": "ちんろうえん",
    "tags": [
      "liyue",
      "location"
    ]
  {
    "en": "Lingshu Courtyard",
    "ja": "霊枢の庭",
    "zhCN": "灵枢庭",
    "pronunciationJa": "れいくのにわ",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Teatree Slope",
    "ja": "老茶樹の丘",
    "zhCN": "古茶树坡",
    "pronunciationJa": "ろうちゃじゅのおか",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Yaodie Valley",
    "ja": "薬蝶の谷",
    "zhCN": "药蝶谷",
    "pronunciationJa": "やくちょうのたに",
    "tags": [
      "liyue",
      "location"
    ]
  {
    "en": "Chizhang Wall",
    "ja": "赤璋の石垣",
    "zhCN": "赤璋城垣",
    "pronunciationJa": "せきしょうのいしがき",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Chiwang Terrace",
    "ja": "赤望台",
    "zhCN": "赤望台",
    "pronunciationJa": "せきぼうだい",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Carp's Rest",
    "ja": "錦落の庭",
    "zhCN": "锦落庭",
    "pronunciationJa": "きんらくのにわ",
    "tags": [
      "liyue",
      "location"
    ]
  },
  {
    "en": "Mt. Xuanlian",
    "ja": "悬练山",
    "zhCN": "灵濛山",
    "pronunciationJa": "けんれんざん",
    "tags": [
      "liyue",
      "location"
    ],
    "note": "The following place name translation is incorrect, so check it on PS5 <a href=\"https://genshin-impact.fandom.com/wiki/Mt._Xuanlian\">"
  }
```

### Object
```Json5
  {
    "en": "Carefree Simulacrum",
    "ja": "仙像",
    "zhCN": "仙像",
    "pronunciationJa": "せんぞう",
    "tags": [
      "liyue",
      "object"
    ],
  }
```

### PR DESCRIPTION
**Pull request by yasuking0304**
## Add/Modified new keywords for Ver.4.1 to 4.5

I referred to here and picked up the missing basic keywords.

https://genshin-impact.fandom.com/wiki

- added version 4.1-4.5 characters
- added version 4.4 drops-boss
- added version 4.4 drops
- added version 4.4 enemies
- added version 4.4 items
- added version 4.4 living beings
- added version 4.4 locations
- added version 4.4 objects
- added version 4.3-4.5 weapons
- added version 4.3-4.5 y-events
- modified japanese notes(delete tag's playable character)
- modified japanese/simplified notes
- added/modified pronunciationJa, and delete notes check comment

### issue

Closes #187

### CODE REVIEW

**Comment by xicri** [comment]
@yasuking0304 Thanks, I will review it!
In the meantime, can you sign the Contributor License Agreement as the bot asked in the above comment?

@Bill-Haku @SleepyAsh0191 Can you review Chinese part of this PR if you have time?

File: dataset/dictionary/locations.json5 [hunk 1]
```diff
@@ -682,6 +682,89 @@
...
+    ja: "錦落の庭",
+    zhCN: "锦落庭",
+    pronunciationJa: "きんらくのにわ",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    // The following place name translation is incorrect, so check it on PS5 <a href=\"https://genshin-impact.fandom.com/wiki/Mt._Xuanlian\">
+    en: "Mt. Xuanlian",
+    ja: "悬练山",
+    zhCN: "灵濛山",
```
Comment (Bill-Haku) [inline dataset/dictionary/locations.json5:764]:
```suggestion
    en: "Mt. Xuanlian",
    ja: "悬练山",
    zhCN: "悬练山",
```

Yes, it's incorrect. But I'm just wondering why check it on **PS5**?

**Review by Bill-Haku — CHANGES_REQUESTED** [review]
Thanks for your contribution @yasuking0304 !

File: dataset/dictionary/y-events.json5 [hunk 2]
```diff
@@ -186,6 +186,151 @@
...
   },
 
+  // v4.5
+  {
+    en: "Blades Weaving Betwixt Brocade",
+    ja: "鋒刃、錦を裁つ",
+    zhCN: "铓锋锦间裁",
+    pronunciationJa: "ほうじんにしきをたつ",
+    tags: [ "event" ],
+    notes: "v4.5 期間限定イベント",
```
Comment (SleepyAsh0191) [inline dataset/dictionary/y-events.json5:196]: Not event name, just code name only

**Review by SleepyAsh0191 — APPROVED** [review]
There are no problems in Chinese part.

**Comment by SleepyAsh0191** [comment]
> There are no problems in Chinese part.

Oh, I forgot the part of bill commented xD

**Review by xicri — CHANGES_REQUESTED** [review]
@yasuking0304 I've added some review comments.

Here are some other requests:

- If there are no official source for `pronunciationJa`, can you add or keep `// TODO Need Check` comment? While it is technically meaningless currently, I will add a property to show if the Japanese pronunciation is reliable one or just our guess.
- Can you fix the CI errors? I guess there are some JSON5 syntax errors and lint errors.

File: dataset/dictionary/characters.json5 [hunk 3]
```diff
@@ -1096,7 +1104,22 @@
...
-    },
+  },
+  {
+    en: "Xianyun",
+    ja: "閑雲",
+    zhCN: "闲云",
+    pronunciationJa: "かんうん",
+    "tags": [ "liyue", "character-main" ],
+    notes: "留雲借風真君/三眼五顯仙人",
+    notesZh: "留云借风真君/三眼五显仙人",
```
[the remark this answers is not in the document]
Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:1115]:
```suggestion
    notesZh: "留云借风真君的俗名",
```

Sure. I think @yasuking0304 knows how to edit it for other items.

File: dataset/dictionary/characters.json5 [hunk 4]
```diff
@@ -1107,6 +1130,8 @@
     ja: "ピンばあや",
     zhCN: "萍姥姥",
     tags: [ "liyue", "character-sub", "sereniteapot" ],
+    notes: "歌塵浪市真君/三眼五顯仙人",
+    notesZh: "歌尘浪市真君/三眼五显仙人",
```
[the remark this answers is not in the document]
Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:1134]:
```suggestion
    notesZh: "雅号为“歌尘浪市真君”",
```

but in fact both 尊称 and 雅号 can be used here in Chinese and maybe 尊称 is a more commonly used word in Chinese. It depends on you @yasuking0304 .

File: dataset/dictionary/characters.json5 (hunk 3 above)
[the remark this answers is not in the document]
Comment (xicri) [reply dataset/dictionary/characters.json5:1115]: @Bill-Haku Thanks!

File: dataset/dictionary/y-events.json5 (hunk 2 above)
Comment (yasuking0304) [reply dataset/dictionary/y-events.json5]: Attached is the revised document.
problem
```
    notes: "v4.5 期間限定イベント",
```
after
```
    notes: "v4.5 バージョンサブタイトル",
```

File: dataset/dictionary/locations.json5 (hunk 1 above)
Comment (yasuking0304) [reply dataset/dictionary/locations.json5]: Sorry. It's a copy and paste mistake
Plogrem
```
    en: "Mt. Xuanlian",
    ja: "悬练山",
    zhCN: "灵濛山",
```
after
```
    en: "Mt. Xuanlian",
    ja: "懸練山",
    zhCN: "悬练山",
```

File: dataset/dictionary/characters.json5 [hunk 5]
```diff
@@ -1096,7 +1104,22 @@
     pronunciationJa: "がんしん",
     variants: {
       en: [ "Geo God", "God of Geo" ],
-    },
```
[the remark this answers is not in the document]
Comment (yasuking0304) [reply dataset/dictionary/characters.json5:1099]: Sorry. It's a correction mistake. Fixed.

[the remark this answers is not in the document]
Comment (xicri) [reply dataset/dictionary/characters.json5:1099]: I guess [the CI error currently raised](https://github.com/xicri/genshin-langdata/actions/runs/8446217592/job/23134664258?pr=188) is caused by this.

File: dataset/dictionary/items.json5 [hunk 6]
```diff
@@ -795,7 +803,7 @@
     en: "Condessence Crystal",
     ja: "萃凝晶",
     zhCN: "萃凝晶",
-    pronunciationJa: "すいぎょうしょう", // TODO Need Check
+    pronunciationJa: "すいぎょうしょう",
```
[the remark this answers is not in the document]
Comment (yasuking0304) [reply dataset/dictionary/items.json5]: Sorry. As you pointed out, I put it back.

File: dataset/dictionary/weapons.json5 [hunk 7]
```diff
@@ -1378,7 +1378,7 @@
     en: "Flowing Purity",
     ja: "純水流華",
     zhCN: "纯水流华",
-    pronunciationJa: "じゅんすいるか", // TODO Need Confirmation
+    pronunciationJa: "じゅんすいりゅうか",
```
[the remark this answers is not in the document]
Comment (yasuking0304) [reply dataset/dictionary/weapons.json5]: Sorry. As you pointed out, I put it back.

File: dataset/dictionary/weapons.json5 [hunk 8]
```diff
@@ -1462,21 +1462,61 @@
...
+    // pronunciationJa: "へきらくのろう",
     tags: [ "weapon", "catalyst" ],
     pinyins: [{ char: "珑", pron: "long2" }],
   },
   {
     en: "Tome of the Eternal Flow",
     ja: "久遠流転の大典",
     zhCN: "万世流涌大典",
-    pronunciationJa: "くおんるてんのだいてん", // TODO Need Check
+    pronunciationJa: "くおんるてんのたいてん",
```
[the remark this answers is not in the document]
Comment (yasuking0304) [reply dataset/dictionary/weapons.json5]: Sorry. As you pointed out, I put it back.

File: dataset/dictionary/weapons.json5 [hunk 9]
```diff
@@ -1462,21 +1462,61 @@
...
     tags: [ "weapon", "catalyst" ],
   },
   {
     en: "Cashflow Supervision",
     ja: "凛流の監視者",
     zhCN: "金流监督",
+    "pronunciationJa": "りるのかんししゃ",
```
[the remark this answers is not in the document]
Comment (yasuking0304) [reply dataset/dictionary/weapons.json5]: As you pointed out, I added 'TODO Need Check'

File: dataset/dictionary/items.json5 [hunk 10]
```diff
@@ -178,8 +178,16 @@
     en: "Crimson Agate",
     ja: "緋紅玉髄",
     zhCN: "绯红玉髓",
+    pronunciationJa: "ひこうぎょくずい",
```
[the remark this answers is not in the document]
Comment (yasuking0304) [reply dataset/dictionary/items.json5]: As you pointed out, I added 'TODO Need Check'

**Comment by yasuking0304** [comment]
Sorry. The lint check was not done and the build was failing, so I took measures.

- Responding to lint indications
- add comma, "}",
- fixed invalid white space
- Added support for Windows for lint check (package.json -> 'lint_windows')

File: dataset/dictionary/living-beings.json5 [hunk 11]
```diff
@@ -165,6 +165,53 @@
...
+    en: "Venerable Jadestone Turtle",
+    ja: "玉璜古ガメ",
+    zhCN: "玉璜古龟",
+    pronunciationJa: "ぎょくごうこがめ",
+    tags: [ "living-being", "liyue" ],
+    notes: "遺瓏埠北部の湾内に生息する巨大亀",
+    notesZh: "一只生活在遗珑埠以北海湾的巨龟",
+  },
+  {
+    en: "Jadestone Turtle",
```
Comment (xicri) [inline dataset/dictionary/living-beings.json5:202]: Maybe you forget to update this to correct English translate when you copied-and-pasted.

**Review by xicri — CHANGES_REQUESTED** [review]

**Comment by yasuking0304** [comment]
Sorry. Duplicate English: Jadestone Turtle
Plogrem
```
    en: "Jadestone Turtle",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```
after
https://genshin-impact.fandom.com/wiki/Velvetfall_Duck
```
    en: "Velvetfall Duck",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```

File: dataset/dictionary/living-beings.json5 (hunk 11 above)
Comment (yasuking0304) [reply dataset/dictionary/living-beings.json5]: Sorry. Duplicate English: Jadestone Turtle
Plogrem
```
    en: "Jadestone Turtle",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```
after
https://genshin-impact.fandom.com/wiki/Velvetfall_Duck
```
    en: "Velvetfall Duck",
    ja: "秋羽鴨",
    zhCN: "秋绒鸭",
```

File: dataset/dictionary/characters.json5 [hunk 12]
```diff
@@ -1650,6 +1676,93 @@
...
+    tags: [ "liyue", "character-sub" ],
+  },
+  {
+    en: "Houzhang",
+    ja: "候章",
+    zhCN: "侯章",
+    pronunciationJa: "こうしょう",
+    tags: [ "liyue", "character-sub" ],
+    notes: "削月築陽真君/三眼五顯仙人",
+    notesZh: "削月筑阳真君/三眼五显仙人"
```
[the remark this answers is not in the document]
Comment (xicri) [reply dataset/dictionary/characters.json5]: @yasuking0304 Can you apply the same change as https://github.com/xicri/genshin-langdata/pull/188#discussion_r1540380408?

File: dataset/dictionary/characters.json5 [hunk 13]
```diff
@@ -1650,6 +1676,93 @@
...
+  },
+  {
+    en: "Jiehu",
+    ja: "接笏",
+    zhCN: "接笏",
+    pronunciationJa: "せっしゃく",
+    tags: [ "liyue", "character-sub" ],
+    notes: "理水畳山真君/三眼五顯仙人",
+    notesZh: "理水叠山真君/三眼五显仙人"
```
[the remark this answers is not in the document]
Comment (xicri) [reply dataset/dictionary/characters.json5]: @yasuking0304 Can you apply the same change as https://github.com/xicri/genshin-langdata/pull/188#discussion_r1540380408?

File: dataset/dictionary/y-events.json5 [hunk 14]
```diff
@@ -186,6 +186,136 @@
...
+  },
+  {
+    en: "The Great Fayz Reaction Debate",
+    ja: "雄弁なフェーイズ作用",
+    zhCN: "弘辩的溢神效应",
+    pronunciationJa: "ゆうべんなフェーイズさよう",
+    tags: [ "event" ],
+    notes: "v4.5 期間限定イベントの子イベント",
```
Comment (xicri) [inline dataset/dictionary/y-events.json5:227]:
```suggestion
    notes: "v4.5 期間限定イベント",
```

**Review by xicri — CHANGES_REQUESTED** [review]

File: dataset/dictionary/items.json5 [hunk 15]
```diff
@@ -1495,7 +1503,7 @@
     en: "Hydro Phantasm Taint-B-Gone",
     ja: "濁水幻霊イチコロリ",
     zhCN: "浊水幻灵克星",
-    pronunciationJa: "だくすいげんれいイチコロリ", // TODO Need Check
+    pronunciationJa: "だくすいげんれいイチコロリ",
```
Comment (xicri) [inline dataset/dictionary/items.json5:1506]: Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "だくすいげんれいイチコロリ", // TODO Need Check
```

File: dataset/dictionary/weapons.json5 [hunk 16]
```diff
@@ -243,7 +243,7 @@
     en: "Wolf-Fang",
     ja: "狼牙",
     zhCN: "狼牙",
-    pronunciationJa: "ろうが", // TODO Need Check
+    pronunciationJa: "ろうが",
```
Comment (xicri) [inline dataset/dictionary/weapons.json5:246]: Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "ろうが", // TODO Need Check
```

File: dataset/dictionary/weapons.json5 [hunk 17]
```diff
@@ -1417,7 +1417,7 @@
     en: "Memory of Dust",
     ja: "浮世の錠",
     zhCN: "尘世之锁",
-    pronunciationJa: "うきよのじょう", // TODO Need check
+    pronunciationJa: "うきよのじょう",
```
Comment (xicri) [inline dataset/dictionary/weapons.json5:1420]: Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "うきよのじょう", // TODO Need Check
```

File: dataset/dictionary/weapons.json5 [hunk 18]
```diff
@@ -1462,7 +1462,7 @@
     en: "Jadefall's Splendor",
     ja: "碧落の瓏",
     zhCN: "碧落之珑",
-    // pronunciationJa: "へきらくのろう", // TODO Need Check
+    // pronunciationJa: "へきらくのろう",
```
Comment (xicri) [inline dataset/dictionary/weapons.json5:1465]: Please revert this if there are no official sources.

```suggestion
    // pronunciationJa: "へきらくのろう", // TODO Need Check
```

**Review by xicri — APPROVED** [review]

**Comment by xicri** [comment]
@yasuking0304 Thanks, I've merged the updates and it should be deployed in several minutes.

**Pull request merged**